functions/src/api/v1/rooms/editRoom.ts

```diff
@@ -47,17 +40,20 @@ export const editRoom = wrap(async (req: Request, res: Response) => {
         id: roomId,
     }
 
-    if (req.body.password) {
-        dataToEdit['password'] = req.body.password
+    if (req.body.password && req.body.password.length > 0) {
```

Comment:
```suggestion
    if (req.body?.password?.length) {
```

or if you prefer `req.body?.password?.length > 0`

Reply:
also, I think `> 0` is not needed